src/services/Initialization.cpp

```diff
@@ -56,19 +56,17 @@ void Initialization::RegisterCoreServices()
     if (ra::services::ServiceLocator::Exists<ra::services::IConfiguration>())
         return;
 
-    auto* pClock = new ra::services::impl::Clock();
-    ra::services::ServiceLocator::Provide<ra::services::IClock>(pClock);
-
-    auto* pFileSystem = new ra::services::impl::WindowsFileSystem();
-    ra::services::ServiceLocator::Provide<ra::services::IFileSystem>(pFileSystem);
-
-    auto* pLogger = new ra::services::impl::WindowsDebuggerFileLogger(*pFileSystem);
-    ra::services::ServiceLocator::Provide<ra::services::ILogger>(pLogger);
-
+    auto pClock = std::make_unique<ra::services::impl::Clock>();
+    auto pFileSystem = std::make_unique<ra::services::impl::WindowsFileSystem>();
+    auto pLogger = std::make_unique<ra::services::impl::WindowsDebuggerFileLogger>(*pFileSystem);
```

Comment:
why change these but not everything in `RegisterServices` below?

Reply:
It didn't catch them, I can change them.